.gitignore

```diff
@@ -41,3 +41,4 @@ testem.log
 .DS_Store
 Thumbs.db
 src/environments
+/src/environments
```

Comment:
Consider using consecutive asterisks for example `**environments` or `**/environments`.